pychromecast/controllers/media.py

```diff
@@ -381,11 +387,27 @@ def disable_subtitle(self):
             "activeTrackIds": []
         })
 
+    def block_until_status_received(self, timeout=None):
+        """
+        Blocks thread until the media status has been updated at least once.
+        The media controller only accepts playback controll commands after a
+        status message has been received.
+
+        If the status has already been received then the method returns
+        immediately.
+
+        :param timeout: a floating point number specifying a timeout for the
+                        operation in seconds (or fractions thereof). Or None
+                        to block forever.
+        """
+        self.status_event.wait(timeout=timeout)
+
     def _process_media_status(self, data):
         """ Processes a STATUS message. """
         self.status.update(data)
 
         self.logger.debug("Media:Received status %s", data)
+        self.status_event.set()
```

Comment:
Good thought. I've slightly reworked/renamed this to reflect the original goal of waiting until a media session is active and ready to accept commands. The event resets itself if the media session id is reset to None.